Add test workflow for Paddle node

MedAliMarz opened the pull request:
This pr includes a workflow to test the Paddle node

Workflow n°203 support:

- Payment: getAll reschedule
- Plan: getAll get
- Product: getAll
- Coupon: create update getAll
- User: getAll

krynble commented:
Hey Dali, on the node `Paddle8` where we reschedule the payment, dates in the past are not accepted.

Can you change this to use a dynamic date in the future? Maybe with Date and Time node something like 5 days in the future?

Thanks!

MedAliMarz commented:
> Hey Dali, on the node `Paddle8` where we reschedule the payment, dates in the past are not accepted.
> 
> Can you change this to use a dynamic date in the future? Maybe with Date and Time node something like 5 days in the future?
> 
> Thanks!

Nice catch, added the Date & Time node and changed date expression.
